Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -405,19 +405,21 @@ def incident_add_comment_command(client, args):
 
 def get_entity_by_id_command(client, args):
     entity_id = args.get('entity_id')
-    url_suffix = f'entities/{entity_id}'
-
-    result = client.http_request('GET', url_suffix, is_get_entity_cmd=True)
+    expansion_id = "98b974fd-cc64-48b8-9bd0-3a209f5b944b"
```

Comment:
why is this hard coded?

Reply:
Take a quick look at this - https://techcommunity.microsoft.com/t5/azure-sentinel/get-entities-for-a-sentinel-incidient-by-api/m-p/1422643